zokrates_js/gulpfile.js

```diff
@@ -0,0 +1,24 @@
+/** general */
```

Comment:
Would be nice to have some comments on what this does, I assume fetching stdlib? What version etc

Reply:
Yes, it's used for fetching and packing standard library into a json file. I added a comment for more clarity.